openreview/conference/invitation.py

```diff
@@ -734,6 +734,10 @@ def __init__(self, conference, note):
                 'forum': note.id,
                 'replyto': None,
                 'readers': reply_readers,
+                'nonreaders': {
+                    'values-dropdown': readers,
+                    'required': False
+                },
```

Comment:
`required` is only available inside content fields :D
```suggestion
                'nonreaders': {
                    'values-dropdown': readers
                },
```